master/getting-started/kubernetes/installation/hardway/standing-up-kubernetes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Standing up Kubernetes
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/installation/hardway/standing-up-kubernetes'
+---
+
+We will install {{site.prodname}} on a Kubernetes cluster. To demonstrate a highly available {{site.prodname}} control plane, we will use five nodes in this guide. This lab walks you through provisioning a Kubernetes cluster in AWS using kubeadm.
+
+## Provision EC2 Nodes
+
+1. Provision five nodes
+    1. Ubuntu 18.04 LTS
+    1. T2.medium
+    1. Ensure the instances are in the same subnet, and security group policy allows them communicate freely with one another.
```

Comment:
> Ensure the instances are in the same subnet

Is this an actual requirement? I've been running masters on a different subnet than the workers and haven't had issues. Plus running in the same subnet would mean it is not ready to run on multiple Availability Zones, which is sad.

Reply:
This guide installs Calico without any overlay, meaning that the nodes need to be on the same subnet for pods to communicate.  If you either

- use an overlay
- have masters without any pods on the pod network (host networked are OK)

then it would be fine to deploy as you say you are running.